test/commands/api/add.test.ts

```diff
@@ -131,7 +133,9 @@ describe('api add', () => {
 
   test
     .do(() => {
-      fs.rmSync(path.resolve('microfrontends', 'mfe1', 'mfe-config.json'))
+      fs.rmSync(
+        path.resolve('microfrontends', 'mfe1', 'public', 'mfe-config.json')
```

Comment:
use constant for mfe-config.json

Reply:
that's just for the test. not sure if it's necessary to use a source constant there.

Reply:
as we already have it defined probably worth to be used

Reply:
just changed `getMfeConfigPath()` from `private` to `public` instead so it can be used there